database-plugins/docs/Database-batchsource.md

```diff
@@ -39,6 +39,11 @@ names are the same when the case is ignored (optional).
 
 **connectionString:** JDBC connection string including database name. (Macro-enabled)
 
+**connectionArguments:** A list of arbitrary string tag/value pairs as connection arguments. This is a
```

Comment:
This description does not makes sense from an end user perspective looking at the plugin. The widgets will do the translation. Can you document this as when users will need to use the connectionArguments. Same comment applies in other places.